contracts/DAO.sol

```diff
@@ -138,26 +140,36 @@ contract DAO {
         dBIOP = dBIOP.sub(staked[msg.sender]);
     }
 
+     //this function has to be present or transfers to the DAO fail silently
+    fallback () external payable {
+        totalStakedAtLastPayment = totalStaked();
+        lastPaymentReceived = block.timestamp;
+            
+        currentClaimPoint = currentClaimPoint+1;
+        totalStakedAtClaimPoint[currentClaimPoint] = totalStaked();
+    }
+
     /**
     * @notice base ETH rewards since last claim
     * @param acc the account to get the answer for
     */
     function bRSLC(address acc) public view returns (uint256) {
```

Comment:
maybe we should use `account` instead of `acc` to be consistent with other functions.

Reply:
nice catch, have done so